src/wipplpy/modules/connection.py

```diff
@@ -0,0 +1,118 @@
+"""
+Define generic objects for accessing WiPPL devices' databases.
+
+TODO: Add a MySQL connection and maybe other connection types as well?
+"""
+
+import logging
+import socket
+from abc import ABC, abstractmethod
+
+import MDSplus as mds
+
+
+class MDSPlusConnection(ABC):
+    """
+    Define generalized routines for accessing MDSplus databases.
+    """
+
+    def __init__(self):
+        self.is_remote_connection = None  # track when remote conn. is made
```

Comment:
I don't think the comment is needed here. Maybe there's a way to have a more understandable variable name like `self.connection_is_remote` instead?